j2k/testData/fileOrElement/literalExpression/hex.kt

```diff
@@ -2,12 +2,13 @@ internal class Test {
     fun test() {
         val i1 = 0x21
         val i2 = 0x33
-        val i3 = -1
+        val i3 = -0x1
         val i4 = 0XCCCC
         val i5 = 0x6fffffff
-        val l1: Long = -1
+        val l1: Long = -0x1
         val l2 = 0xffffffffL
         val l3 = 0xffffffffffL
-        val l4 = -1
+        val l4 = -0x1
```

Comment:
`L` is missing after conversion, so result type of `l4` would be `Int` instead of `Long`